lib/importer/attach_files_to_work.rb

```diff
@@ -0,0 +1,14 @@
+# Called in 'object_factory'
+module Importer
+  class AttachFilesToWork < AttachFilesToWorkJob
```

Comment:
Rather than doing this, would it be possible to coerce the input data to an Array of files (with only one element)?  That means that any upstream changes won't have to be ported down to Hyku?

Reply:
I see how overriding upstream changes could become undesirable - I'll work on this thanks!